dremio_client/model/data.py

```diff
@@ -744,7 +744,7 @@ def __init__(self, token=None, base_url=None, flight_endpoint=None, ssl_verify=T
         )
 
     def query(self):
-        return self.sql("select * from {}")
```

Comment:
I would create a method in `Dataset` called `get_table` or so and do the join in there and call that rather than doing the path join from inside the query method

Reply:
this better ?